openupgrade_scripts/scripts/hr_timesheet/14.0.1.0/noupdate_changes.xml

```diff
@@ -19,6 +19,9 @@
     <field name="category_id" ref="base.module_category_services_timesheets"/>
     <!-- <field name="implied_ids" eval="[(4, ref('hr_timesheet.group_hr_timesheet_approver')), (4, ref('hr.group_hr_user'))]"/> -->
   </record>
+  <record id="project.group_project_manager" model="res.groups">
```

Comment:
Please comment the whole record instead of just the field, so no empty write is done at all.

Reply:
Done